src/main/scala/org/hammerlab/guacamole/commands/GermlineAssemblyCaller.scala

```diff
@@ -74,17 +75,19 @@ object GermlineAssemblyCaller {
           .getPartitioner(mappedReads, args.assemblyWindowRange)
           .partition(loci.result(contigLengths))
 
-      val genotypes: RDD[CalledAllele] = discoverGermlineVariants(
```

Comment:
A theme that has emerged for me is that I like only having one thing happening on a given line; scala is *so* dense, and lines are so much wider than they are tall (in terms of pixels on screen), that things tend to be clearer the more lines they are split over, at least starting from where we are today.

So here, I like naming a variable on one line, opening the function call on the next, specifying each arg on its own line, and closing the function scope with a `)` line.

Again, not saying you have to do things this way, but if you are at worst neutral on it, I really prefer it. Many more examples to come.